test/opt/pass_fixture.h

```diff
@@ -89,11 +89,23 @@ class PassTest : public TestT {
   // disassembly string and the boolean value from the pass Process() function.
   template <typename PassT, typename... Args>
   std::tuple<std::string, opt::Pass::Status> SinglePassRunAndDisassemble(
-      const std::string& assembly, bool skip_nop, Args&&... args) {
+      const std::string& assembly, bool skip_nop, bool do_validation,
```

Comment:
default do_validation to 'true'?

Reply:
I didn't want to make that choice for other tests. Its possible they are using invalid IR for their testcases.